AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/capture/DefaultCameraCaptureSource.swift

```diff
@@ -12,7 +12,16 @@ import UIKit
 
 @objcMembers public class DefaultCameraCaptureSource: NSObject, CameraCaptureSource {
     public var videoContentHint: VideoContentHint = .motion
+    
+    /// Expose current capture device's torch availability
+    public var isTorchAvailable: Bool {
```

Comment:
You can replace `if let captureDevice = captureDevice, captureDevice.hasTorch, captureDevice.isTorchAvailable` on line 83 with `if let captureDevice = captureDevice, torchAvailable()`